Source/WebCore/accessibility/mac/WebAccessibilityObjectWrapperMac.mm

```diff
@@ -2464,6 +2476,44 @@ - (id)accessibilityHitTest:(NSPoint)point
         hit = self;
 
     return NSAccessibilityUnignoredAncestor(hit);
+
+}
+
+- (void)accessibilityHitTestResolvingRemoteFrame:(NSPoint)point callback:(void(^)(NSString *))callback
+{
+    if (!AXObjectCache::clientIsInTestMode())
+        return;
```

Comment:
Do we want all flow paths to call callback?

Reply:
Yeah, it probably can't hurt. I mostly have this to prevent non-testing clients from using `accessibilityHitTestResolvingRemoteFrame`, so we should just give those empty strings as parameters.